specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -812,25 +843,254 @@ model ConversationRequestStatistics extends RequestStatistics {
 @added(Versions.v2023_04_01)
 model ConversationItemLevelTiming extends AudioTiming {}
 
+@added(Versions.v2023_04_15_preview)
+@doc("Task definition for conversational sentiment.")
+model AnalyzeConversationalSentimentTask extends AnalyzeConversationLROTask {
+  @doc("discriminator kind")
+  kind: AnalyzeConversationLROTaskKind.ConversationalSentimentTask,
+
+  @doc("temp doc")
```

Comment:
Better to add a suppression, since we do scan for those (or will, anyway). Better reminder than a valid - but otherwise useless - `@doc()`.